### PR TITLE
CSR tests

### PR DESCRIPTION
**Pull request by inteldimitrius**
# Description
Tests for CSR table added.

### CODE REVIEW

File: cpp/oneapi/dal/table/test/csr_serialization.cpp [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    const std::int64_t row_count{ 8 };
+    const std::int64_t column_count{ 8 };
+    const std::int64_t element_count{ 9 };
+
+    csr_table init_table{ array<float>::wrap(data, element_count),
+                          array<std::int64_t>::wrap(column_indices, element_count),
+                          array<std::int64_t>::wrap(row_indices, row_count + 1),
+                          column_count,
+                          sparse_indexing::zero_based };
+    std::cout << "TABLE successfully created" << std::endl;
```
Comment (ethanglaser) [inline cpp/oneapi/dal/table/test/csr_serialization.cpp:94]: I see this in just one test, I would probably remove, or add to all if its necessary.

**Review by ethanglaser — APPROVED** [review]
Looks good to me but will also defer to Vika and those more familiar with CSR here.

File: cpp/oneapi/dal/table/test/csr_serialization.cpp (hunk 1 above)
Comment (samir-nasibli) [reply cpp/oneapi/dal/table/test/csr_serialization.cpp:94]: please remove debug print.

**Pull request merged**